src/test/java/uk/gov/pay/card/it/resources/CardIdResourceITest.java

```diff
@@ -23,37 +23,34 @@
     public DropwizardAppRule<CardConfiguration> app = new DropwizardAppRule<>(
             CardApi.class
             , resourceFilePath("config/config.yaml")
+            , config("server.applicationConnectors[0].port", "9800")
+            , config("server.adminConnectors[0].port", "9801")
```

Comment:
This is causing issues running concurrent builds due to being unable to bind to port, already in use.